SportsHub/SportsHub.Api/Exceptions/ExceptionHandler.cs

```diff
@@ -1,20 +1,55 @@
 ﻿using System.Net;
+using SportsHub.Api.Exceptions.CustomExceptionModels;
 
 namespace SportsHub.Api.Exceptions;
 
 public class ExceptionHandler: IMiddleware
 {
+    private CustomException _exception;
     public async Task InvokeAsync(HttpContext context, RequestDelegate next)
     {
         try
         {
             await next(context);
         }
-
-        catch (Exception e)
+        catch (BusinessLogicException ex)
+        {
```

Comment:
Do you need these 2 catch blocks, or is it possible to combine them?

Reply:
Yes i forgot to change it there . With my abstract class i can catch with CustomException and then in my private method to set which one should be .